common/Root.tsx

```diff
@@ -66,6 +67,7 @@ export default class Root extends Component<Props, State> {
         <Route path="/ens" component={ENS} />
         <Route path="/sign-and-verify-message" component={SignAndVerifyMessage} />
         <Route path="/pushTx" component={BroadcastTx} />
+        <Route path="/status" component={TransactionStatus} />
```

Comment:
To keep with v3, path should be `/check-tx-status`. Looks like `LegacyRoutes` below was already mapped out with that in mind.